wesnoth-units.po

```diff
@@ -6835,6 +6775,13 @@ msgid ""
 "inevitably the most shrewd and perceptive among his peers and has won the "
 "respect of his brethren through leadership rather than coercion."
 msgstr ""
+"Běžnou mylnou představou o hierarchii skřetích klanů je ta, že skřeti "
+"oceňují pouze hrubou sílu. I když vůdcem menšího skřetího vojska může být "
+"ten nejsilnější, vládce velkého kmene musí zvládat obtíže při potlačování "
+"vnitřních svárů a bojů, což je něco, čeho nelze pouze silou dosáhnout. "
+"Takový skřet pak nevyhnutelně patří mezi nejchytřejší a nejvnímavější mezi "
+"svými vrstevníky, který si respekt svých bratří získal spíše vedením než "
```

Comment:
který by patřilo za Takový skřet - tím by se ale změnil smysl věty
nechal bych tam to a, které je i v předloze
```suggestion
"svými vrstevníky a respekt svých bratří si získal spíše vedením než "
```